package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@google/cloud-trace",
+  "name": "@google-cloud/trace-agent",
```

Comment:
Yup, all future releases will be published to `@google-cloud/trace-agent`. We will be deprecating `@google/cloud-trace`.